src/hermes/commands/harvest/git.py

```diff
@@ -320,7 +320,8 @@ def harvest_git(click_ctx: click.Context, ctx: HermesHarvestContext):
 
         '@type': "SoftwareSourceCode",
         'contributor': [contributor.to_codemeta() for contributor in git_contributors._all],
-    }, branch=git_branch)
+    })
```

Comment:
Why did you remove the `branch` tag here? I know, it's duplicate information. However, it might still be vital for later auditioning (e.g., if some other source overrides parts of the contributor information here).

Reply:
```suggestion
    }, git_branch=git_branch)
```

Re-add the metadata metadata.